src/main/java/LottoApplication.java

```diff
@@ -0,0 +1,9 @@
+import controller.LottoController;
+
+public class LottoApplication {
+
+    public static void main(String[] args) {
+        new LottoController().run();
+    }
```

Comment:
`new LottoController().run()`으로는 보통 사용하지 않기 때문에 아래처럼 수정하시면 좋아보여요~
```suggestion
    public static void main(String[] args) {
        LottoController lottoController = new LottoController();
        lottoController.run();
    }
```